### ISSUE TITLE
wxInfoBar icon to large on High DPI screen

### ISSUE THREAD

**Issue by sz5000** [wxWidgets/wxWidgets#23031]
On my High DPI Screen the standard Icon used in the wxInfoBar is to large. My display uses a scaling factor of 2.5. It's a 4k display. Seems that the display factor is applied twice.

```
wxInfoBar* infoBar = new wxInfoBar(this);
infoBar ->ShowMessage(_("Unsupported PDF feature"), wxICON_INFORMATION);
```
The icon is loaded by `wxWindowsArtProvider::CreateBitmap` in file artmsw.cpp.

m_sizeDefault seems to be 40,40 which is 16 x 2.5 and so 40,40 is already scaled. `GetPreferredBitmapSizeAtScale` returns 3.0 and thus `DoGetPreferredSize` returns 120,120. Size 40,40 comes from `wxArtProvider::GetNativeSizeHint(client)`.

Is this behavior intentional?

![image](https://user-images.githubusercontent.com/33447587/207030534-fa8799c5-76e0-4b97-90f4-7966b1548aaf.png)

**Comment by vadz** [comment]
I do see the problem and I think it's due to not setting the scaling factor of the bitmap we create correctly: when creating a 40x40 bitmap, it should already have the factor of 2.5 and not 1 as it was the case. So I've fixed this now in #23038, please let me know if you still see any problems.

And thanks for reporting this!

**Comment by sz5000** [comment]
Seems to solve this issue. The icon now has a normal/expected size. Thanks.
